src/components/PeopleTable/PeopleTable.jsx

```diff
@@ -0,0 +1,236 @@
+          Sort by
+        </Text>
+        <Select className="sort-options" width="30%">
+          <option value="option1">Name: A-Z</option>
+          <option value="option2">Name: Z-A</option>
+        </Select>
+      </Flex>
+    </Flex>
+  );
+};
+
+/* eslint-disable react/jsx-key */
+const StyledHeader = ({ headerGroups }) => {
+  return headerGroups.map(headerGroup => (
+    <Tr className={styles['table-head']} {...headerGroup.getHeaderGroupProps()}>
+      {headerGroup.headers.map(column => (
+        <Th {...column.getHeaderProps()} userSelect="none" color="white" bgColor="ochGrey">
+          <Flex alignItems="center">
+            {column.render('Header')}
+            <ChevronDownIcon ml={1} w={4} h={4} />
+          </Flex>
+        </Th>
+      ))}
+    </Tr>
+  ));
+};
+/* eslint-enable react/jsx-key */
+
+const StyledFooter = ({ rowCount, pageIndex, pageSize, setPageSize }) => {
+  const rowText = () => {
+    const left = pageIndex === 0 ? 1 : (pageIndex + 1) * pageSize;
+    const right = Math.min(rowCount, pageSize * (pageIndex + 1));
+    return `${left}-${right}`;
+  };
+  return (
+    <Box
+      className={styles['footer-container']}
+      backgroundColor="ochGrey"
+      color="white"
+      style={{ margin: 0 }}
+    >
+      <Flex justifyContent="space-between" m={4} alignItems="center">
+        <Flex alignItems="center">
+          <Text flexShrink="0">Show rows per page: </Text>{' '}
+          <Select
+            backgroundColor="white"
+            color="ochGrey"
+            ml={2}
+            w={32}
+            value={pageSize}
+            onChange={e => {
+              setPageSize(Number(e.target.value));
+            }}
+          >
+            {rowsPerPageSelect.map(rowVal => (
+              <option key={rowVal} value={rowVal}>
+                {rowVal}
+              </option>
+            ))}
+          </Select>
+        </Flex>
+
+        <Flex alignItems="center">
+          <Text flexShrink="0" mr={8}>
+            <Text as="span">{rowText()}</Text>
+            <Text as="span" color="#cbd5e0">
+              {' '}
+              of {rowCount}
+            </Text>
+          </Text>
+          <Tooltip label="Previous Page">
+            <IconButton
+              background="transparent"
+              color="white"
+              icon={<ChevronLeftIcon h={6} w={6} />}
+            />
+          </Tooltip>
+          <Tooltip label="Next Page">
+            <IconButton
+              background="transparent"
+              color="white"
+              icon={<ChevronRightIcon h={6} w={6} />}
+              ml={4}
+            />
+          </Tooltip>
+        </Flex>
+      </Flex>
+    </Box>
+  );
+};
+
+const PeopleTable = ({ variant, peopleData }) => {
+  const columns = useMemo(() => cellStructure, []);
+  // Memoizing data
+  const data = useMemo(() => peopleData, []);
+
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    page,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'page' is assigned a value but never used.